src/diffusers/configuration_utils.py

```diff
@@ -358,8 +359,17 @@ def load_config(
                 "`self.config_name` is not defined. Note that one should not load a config from "
                 "`ConfigMixin`. Please make sure to define `config_name` in a class inheriting from `ConfigMixin`"
             )
-
-        if os.path.isfile(pretrained_model_name_or_path):
+        # Custom path for now
+        if dduf_entries:
+            if subfolder is not None:
+                config_file = os.path.join(pretrained_model_name_or_path, subfolder, cls.config_name)
+            else:
+                config_file = os.path.join(pretrained_model_name_or_path, cls.config_name)
```

Comment:
It feels weird to me to use `pretrained_model_name_or_path` and `subfolder` to generate a DDUF entry. These two parameters refers to paths on a system. But when dduf entries are passed, the entry names are not based on paths on the system.

Maybe `subfolder` can be useful to load from a subfolder in the DDUF archive (but I haven't spotted where this is done). For `pretrained_model_name_or_path` it seems to be explicitly set to `""` below to make it work here by ignoring the value in `os.path.join`. That feels like a weird behavior to me (I'd just not used these values here).

Reply:
Also, paths inside a DDUF file must always be `"/"`-separated so this code could produce an error with Windows paths `"\\"`

Reply:
Actually, `pretrained_model_name_or_path` will take for value "tokenizer" or "text_encoder" for example. We discard the paths on the system but only keep the structure of the folder.

Reply:
> Actually, pretrained_model_name_or_path will take for value "tokenizer" or "text_encoder" for example. We discard the paths on the system but only keep the structure of the folder.

Oh ok, good! But then `subfolder` is not needed at this stage? Since DDUF files can only have 1 level of directory.

Reply:
Make sense, we only allow for 1 level of directory, then it shouldn't be necessary.

Reply:
fixed it